examples/gimbal_raw/gimbal_raw.cpp

```diff
@@ -0,0 +1,114 @@
+//
+// Connect to and control a gimbal manager, using the "raw" GimbalManager API
```

Comment:
I don't get this one. What do you mean with `raw`?

Reply:
@julianoes I mean `raw` in the same way as `MissionRaw`, in the sense that the GimbalManager API is low level control, versus the (to be written) higher level `Gimbal` API.

Reply:
Ok, and what is that needed for?